src/app.py

```diff
@@ -34,12 +34,27 @@
     #print(our_questions)
 
 
-    # Note: Implement argparse to parse `--discord` argument if set.
-    #       After parsing arguments, we will do something like the following:
-    #
-    # if "discord mode":
-    #     bot = DiscordUtil(DISCORD_AUTH_TOKEN)
-    # else:
-    #     bot = StandaloneUtil()
-    #
+    command_parser = argparse.ArgumentParser(
+        prog='TrainingWheels Bot',
+        usage='app.py [--discord]',
+        formatter_class = argparse.RawDescriptionHelpFormatter,
+        description='''\
+    TrainingWheels Bot.
+
+    A Multi-functional bot built for the TrainingWheels discord server.
+    Primary function is to set, track, and record users' LeetCode challenge progress.
+    -----------------------------------------
+    Default == Standalone version, locally interactable via CLI''')
+
+    command_parser.add_argument('--discord',
+        action='store_true',
+        help='run the app in Discord mode, no arguments are needed for this command.')
+
+    args = vars(command_parser.parse_args())
+    if args['discord']:
```

Comment:
You should just be able to do the following
```
args = command_parser.parse_args()
if args.discord:
```
The way you have it works but it's extra work. Check https://docs.python.org/3/library/argparse.html#core-functionality for more details.

Reply:
Thanks for the clarification, updated.